examples/minimal/client/rust/src/main.rs

```diff
@@ -0,0 +1,96 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! This is a minimal Rust client. It does a simple request, containing a string
+// to an Oak application and expects a response with a string in it.
+//
```

Comment:
Should all the comment lines here be `//!`?

Reply:
I don't know what //! does :) Should we?